m3u8-cmdline/m3u8.py

```diff
@@ -0,0 +1,80 @@
+import subprocess
+import os
+import platform
+import json
+import argparse
+
+class config_info:
+  def __init__(self, system):
+    if system == 'Windows':
+      self.directory = os.getenv('PROGRAMDATA') + '\\m3u8-streamer\\config\\'
+    elif system == 'Linux':
+      self.directory = os.path.expanduser('~') + '/.config/m3u8-streamer/config/'
+    self.os = system
+    self.path = self.directory + 'config.json' 
+
+config = config_info(platform.system())
+
+
+def Initialize(args):
+  data = {"streamkey":"",
+          "ffmpeg":"ffmpeg -loglevel warning -reconnect 1 -reconnect_at_eof 1 -reconnect_delay_max 10 -i {0} -codec:a aac -c:v copy -f flv rtmp://ingest.angelthump.com:1935/live/{1}"}
+  
+  if not os.path.isdir(config.directory):
+    try:
+      os.makedirs(config.directory)   
+    except Exception as e:
+      print(e)
+  if not os.path.isfile(config.path):
+    try:
+      with open(config.path, 'w') as config_file: json.dump(data, config_file)
+    except Exception as e:
+      print(e)
+
+
+
+def readConfig(args):
```

Comment:
I think this function can be simplified to something like:
```
if args.streamkey is not None:
  write it to config and set streamkey to args.streamkey
else
  read config and set streamkey to args.streamkey
``` 
moving the file handling to inside the conditional branch. If a user provides the streamkey arg, it should always update the config (I think you do this already) and be used in the current call, no reading necessary. If not provided, reading is all that is necessary.